numba/core/pylowering.py

```diff
@@ -188,21 +194,29 @@ def lower_assign(self, inst):
         elif isinstance(value, ir.Yield):
             return self.lower_yield(value)
         elif isinstance(value, ir.Arg):
+            # TODO: check why value.name may not exist
```

Comment:
This TODO apply still? What's the context that triggers it?

Reply:
It no longer applies. To be removed.

Reply:
removed in https://github.com/numba/numba/pull/5922/commits/d328ac507d7c05fa731e856972f22344e7151b34